test/Mono.Linker.Tests.Cases/DynamicDependencies/DynamicDependencyMemberSignatureWildcard.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Runtime.CompilerServices;
+using System.Diagnostics.CodeAnalysis;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.DynamicDependencies
+{
+	[LogContains ("IL2033: Unresolved member '*' in DynamicDependencyAttribute")]
+	[SetupCompileBefore ("library.dll", new[] { "Dependencies/DynamicDependencyMethodInAssemblyLibrary.cs" })]
+	public class DynamicDependencyMemberSignatureWildcard
+	{
+		public static void Main ()
+		{
+			Dependency ();
+		}
+
+		[Kept]
+		[DynamicDependency ("*", "Mono.Linker.Tests.Cases.DynamicDependencies.Dependencies.DynamicDependencyMethodInAssemblyLibrary", "library")]
```

Comment:
I thought that we said we would not support wildecards - instead we have the DynamicallyAccessedMemberTypes for that purpose... right?

Reply:
This test is just checking the warning.

Reply:
Sorry - I missed that.